README.md

```diff
@@ -19,14 +22,14 @@ Configuration parsing should be easy as it sounds - Hence;
 
 import zio.{App, ZIO}
 import zio.config._
+import zio.console.Console
 
 object ReadConfig extends App {
-  case class Prod(ldap: String, dburl: Option[String], regions: List[Int])
```

Comment:
List support to be added later

Reply:
Yes, list is a prioritised feature. Just a matter, how do user need to express it.

`opt(config)` makes sense intuitively. Optionally read all the values associated with the `config`. 

but what exactly is `list(config)`. Immediate thinking is reading all the values within that config as a list?
So library will assume the following. It will be a delimited string. But is that the only representation? 

`string("possibly-a-list").mapEither(t => t.split(",").toList)(_.mkString(","))` is the implementation. But is that a function in the library?

Reply:
In other words,
`list(int("x"))` could possibly mean the following

1. Read the integer first, and then convert it to a list.
2. Or (as library used to do), read the string first, delimit split, convert each of them to an integer.